GeeksCoreLibrary/Modules/OpenCatalogInterface/Models/CxmlPunchOutSetupResponseModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Xml.Serialization;
+
+namespace GeeksCoreLibrary.Modules.OpenCatalogInterface.Models;
+
+[XmlRoot(ElementName = "cXML")]
+public class CxmlPunchOutSetupResponseModel
+{
+    [XmlElement(ElementName = "Response")]
+    public CxmlPunchOutResponseModel Response { get; set; } = new();
+
+    [XmlAttribute(AttributeName = "version")]
+    public string Version { get; set; }
+
+    [XmlAttribute(AttributeName = "lang", Namespace = "http://www.w3.org/XML/1998/namespace")]
```

Comment:
Yes, that is because `lang` is a special attribute in XML. It always needs a namespace and it's written as `<element xml:lang="x">` instead of `<element lang="x">`. Adding a namespace tells the serializer to use the `xml:lang` syntax.

Reply:
Alright, but I noticed other models have a `lang` attribute as well, such as `CxmlModel` and `Comment` (among others), but those don't have a namespace. Is this the only exception that needs a namespace?

Reply:
Oh you are correct, did not notice that. Maybe I am misunderstanding something then. I will have to look it up and get back to you.

Reply:
I have added the namespace to all lang properties now, to make it consistent.